src/safe_apps/views.py

```diff
@@ -42,14 +42,14 @@ def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
     def get_queryset(self) -> QuerySet[SafeApp]:
         queryset = SafeApp.objects.filter(visible=True)
 
-        network_id = self.request.query_params.get("chainId")
-        if network_id is not None and network_id.isdigit():
-            queryset = queryset.filter(chain_ids__contains=[network_id])
+        chain_id = self.request.query_params.get("chainId")
+        if chain_id is not None and chain_id.isdigit():
+            queryset = queryset.filter(chain_ids__contains=[chain_id])
 
-        host = self.request.query_params.get("clientUrl")
-        if host is not None:
+        client_url = self.request.query_params.get("clientUrl")
+        if client_url is not None and client_url:
```

Comment:
Wouldn't it be enough to have 

```python
if client_url:
```

Probably the same can be done above too:

```python
if chain_id.isdigit():
```

Reply:
for the `client_url` it works, thanks for the tip.

I think you cannot call `isdigit` method on `None`, but I can specify an empty string as the default value in param getter, wdyt?

Reply:
I'd say to leave both conditions in (ie.: not have the default for now)